packages/@magic-sdk/react-native-bare/src/react-native-webview-controller.tsx

```diff
@@ -80,18 +83,41 @@ export class ReactNativeWebViewController extends ViewController {
   /* istanbul ignore next */
   public Relayer: React.FC<{ backgroundColor?: string }> = (backgroundColor) => {
     const [show, setShow] = useState(false);
+    const [mountOverlay, setMountOverlay] = useState(true);
     const isConnected = useInternetConnection();
 
     useEffect(() => {
       this.isConnectedToInternet = isConnected;
     }, [isConnected]);
 
     useEffect(() => {
+      // reset lastMessage when webview is first mounted
+      AsyncStorage.setItem('lastMessageTime', '');
       return () => {
         this.isReadyForRequest = false;
       };
     }, []);
 
+    useEffect(() => {
+      this.eventEmitter.addListener(MSG_POSTED_AFTER_INACTIVITY_EVENT, async (message) => {
```

Comment:
Can this be done via a React State?

Reply:
No because the emit is done on _post which is not part of the component